inttest/airgap/airgap_test.go

```diff
@@ -36,7 +36,7 @@ spec:
 const etcHosts = `
 127.0.0.8 docker.io
 127.0.0.8 gcr.io
-127.0.0.8 k8s.gcr.io
+127.0.0.8 registry.k8s.io
```

Comment:
I'd also leave `k8s.gcr.io` into the "block" list to be sure we block any accidental access to it too

Reply:
I'd like to file a separate PR for this, as this is a backport, which wouldn't be on par with the original PR. Alternatively, I can add a commit here, which we then cherry-pick back to main.

Reply:
good point. separate PR makes sense.